website/content/en/events/2024-cloud-native-sustainability-week.md

```diff
@@ -40,6 +40,7 @@ After the successful [Sustainability Week 2023](https://tag-env-sustainability.c
 | 10 | Kuala Lumpur, Malaysia | Oct. 21 | TBD | Mageshwaran Sekar, Khairul Anuar, Ahmed Zidan
 | 11 | Stuttgart, Germany | TBD| TBD | Danijel Soldo, Stefan Bergstein
 | 12 | Paris, France | Oct. 15 | TBD | Smaine Kahlouch [Cloud Native Paris](https://community.cncf.io/paris/)
+| 13 | Brighton, UK | Oct. 16 | [Link](https://www.meetup.com/gsf-brighton/events/303193268) | Adam Newman, [Green Software Brighton](https://greensoftwarebrighton.co.uk)
```

Comment:
@anewmandev thanks for the PR! Can you please also update the same file for different languages? Here's an example from the earlier PR: https://github.com/cncf/tag-env-sustainability/pull/505/files

Reply:
Hi Kristina. I have done that now, I acidentally put a couple in 2023 so reverted and corrected. Sorry for the erroneous commits. @guidemetothemoon

Reply:
@anewmandev LGTM, thanks a lot!